plugins/janus_recordplay.c

```diff
@@ -2618,6 +2677,98 @@ static void *janus_recordplay_playout_thread(void *data) {
 				}
 			}
 		}
+		if(data) {
+			if(data == session->dframes) {
+				/* First packet, send now */
+				/* Data recording stores recording timestamp in first 8 bytes - it follows the frame ts monotonically.
+					invariant: when = data->ts + when(initial packet) */
+				gint64 when = 0;
+				int len = data->len;
+				int offset = data->offset;
+				fseek(dfile, data->offset, SEEK_SET);
+				bytes = fread(&when, sizeof(gint64), 1, dfile);
+				when = ntohll(when); // NOTE: not currently used - playback is interested in actual data packets.
+				offset += sizeof(gint64);
+				len -= sizeof(gint64);
+
+				/* Read data packet */
+				fseek(dfile, offset, SEEK_SET);
+				bytes = fread(buffer, sizeof(char), len, dfile);
+				JANUS_LOG(LOG_INFO, "Sending data packet at rtp_timestamp = %lu, timestamp = %lu, delta = %lu\n", (data->ts), when, when - data->ts);
```

Comment:
This shouldn't be `LOG_INFO`, but `LOG_HUGE` (unless it's just there for your own debugging?).

Reply:
Yup, just for my debugging. Corrected.